modules/bcf/lib/open_project/bcf/engine.rb

```diff
@@ -51,13 +51,12 @@ class Engine < ::Rails::Engine
                    'bcf/issues': %i[index upload prepare_import configure_import perform_import]
       end
 
-      menu :project_menu,
-           :bcf,
-           { controller: '/bcf/issues', action: :index },
-           caption: :'bcf.label_bcf',
-           param: :project_id,
-           icon: 'icon2 icon-backlogs',
-           badge: 'bcf.experimental_badge'
+      rename_menu_item :project_menu,
```

Comment:
nice! Maybe we could even extend it to take a different URL to call? It would be great if this menu item and the view "All open" opens the grid view.